### ISSUE TITLE
[Usable - Azure Databases Extension - Results tab]: Screen reader announces the role “toolbar” information twice for the “Result view” toolbar.

### ISSUE THREAD

**Issue by ChandiniSmiley** [microsoft/vscode-cosmosdb#2668]
### **User Experience:** 

Visually impaired users who depend on screen reader for navigation will not understand the exact purpose of the control if incorrect name is defined and will find it difficult to perform the action related to the control. 

**Note:** Login Credentials must not be provided within the bug. 

### **Pre-requisite:** 

Azure database should be installed from extensions in VS code. 

### **Repro Steps** 

1. Open VS Code and navigate to azure databases button present on right pane. 
2. Navigate to anyone of the resource and expand it. Expand a database. Right click on any collection and select "Open Query editor(Preview)". 
3. Query editor screen appears. Run any sample query. Data in table appears under results tab. 
4. Navigate through all controls of the results tab including running the query. 
5. Verify if the role 'toolbar' itself announced twice for 'Result view' toolbar. 

### **Actual Result:** 

Screen reader Narrator announces the role “toolbar” information twice for the “Result view” toolbar. 

Observation : Same issue is repro with both NVDA and JAWS. Both the screen readers announces as “Result view toolbar tool bar”. 

### **Expected Result:** 

Screen reader Narrator should announce the role “toolbar” information only once for the “Result view” toolbar.

### **Reference links:** 
 

- External Bug Process: If this bug belongs to external team, mark it as resolved/done and assign it back to the tester with notes on where to file/route the bug. For more information, please use this [External Bug Process (sharepoint.com)](https://microsoft.sharepoint.com/teams/cea11y/SitePages/External-Bug-Process.aspx?xsdata=MDN8MDF8fDE1MThmM2JlZGFjMDRlZDhiZmViYTUxMDA2M2NiZWI3fDcyZjk4OGJmODZmMTQxYWY5MWFiMmQ3Y2QwMTFkYjQ3fDF8MHw2Mzc3NDEyNTM2MTEzNzIwNzR8R29vZHxWR1ZoYlhOVFpXTjFjbWwwZVZObGNuWnBZMlY4ZXlKV0lqb2lNQzR3TGpBd01EQWlMQ0pRSWpvaUlpd2lRVTRpT2lJaUxDSlhWQ0k2TVRKOQ%3D%3D&sdata=MnlpSXlVMnpGT3cvZ0xJNnFWUWZWTGt6dmo1VkJMZFpIZHBvUUduNDhyMD0%3D&ovuser=72f988bf-86f1-41af-91ab-2d7cd011db47%2cv-kgeetala%40microsoft.com). “HCL Staff should not log any third-party external bugs. Should be routed to EDAD team”. 
- Please reach out to [C&AI Teams channel](https://apc01.safelinks.protection.outlook.com/?url=https://teams.microsoft.com/l/team/19%253a3cd071821b694f11b054d878453cd75f%2540thread.skype/conversations?groupId%3D30e3f968-15dd-4da2-bd8e-85f27e4af2c3%26tenantId%3D72f988bf-86f1-41af-91ab-2d7cd011db47&data=02%7c01%7cAklavyaM%40hcl.com%7c24e89c6ac26f42cfe40c08d7667e49f3%7c189de737c93a4f5a8b686f4ca9941912%7c0%7c0%7c637090564906844600&sdata=22JjBMoA1ID/KnYqFa/OB%2BrkTaLh5YYsGBd/7Bm7u5w%3D&reserved=0) for any process related queries.

### **Test Environment:**   

- OS Version: 24H2 (OS Build 26100.3476) 
- Visual Studio Code Version: 1.99.0 (user setup) 
- Tool: Accessibility Insights for Win32 
- Azure Databases for VS Code (Preview) Version : 0.25.2 
- Matrix:Win32+Narrator

https://github.com/user-attachments/assets/e5a15dc6-322d-4308-bd19-b18a07c04bec

**Comment by ChandiniSmiley** [comment]
#A11yUsable;#Accessibility;#Benchmark;# GH-HCL-AzureDatabasesExtension-Apr2025;#HCL;#Win32;

**Comment by ChandiniSmiley** [comment]
@mkrueger , As we checked in the below version, the issue is still repro. Hence reactivating the bug.

**Test Environment:**

- OS Version: 24H2 (OS Build 26100.3476)
- Visual Studio Code Version: 1.99.0 (user setup)
- Tool: Accessibility Insights for Win32
- Azure Databases for VS Code (Preview) Version : 0.25.2
- Matrix:Win32+Narrator
